config/locales/fr.yml

```diff
@@ -456,8 +456,9 @@ fr:
             announcement_help: Le texte saisi ici sera affiché juste en dessous des informations de la concertation.
             duration: Durée
             emitter: Organisateur
-            emitter_logo: Logo de l'organisateur de la consultation
+            emitter_logo: Choisissez un nouveau logo pour l'organisateur de la consultation
             emitter_name: Nom du logo
+            emitter_logo_select: Ou importez un nouveau logo
```

Comment:
Can you update this translation for something more explicit ? 
Like:
`Récupérer un logo d'organisateur déjà existant sur cette plateforme`